provision/ubuntu/install.sh

```diff
@@ -38,9 +38,8 @@ if [ "${NETNEXT}" == "true" ]; then
     sudo rm $(which mount.vboxsf)
 fi
 
-# Disable unattended-upgrades to prevent it from holding the dpkg frontend lock
-sudo systemctl disable unattended-upgrades.service
-sudo systemctl stop unattended-upgrades.service
+# Remove unattended-upgrades to prevent it from holding the dpkg frontend lock
+sudo apt-get remove -y unattended-upgrades
```

Comment:
Might be safer to use `apt-get remove --purge` here to make sure the systemd service files (and any other remaining config) are indeed also removed?

```suggestion
sudo apt-get remove --purge -y unattended-upgrades
```

Reply:
Good point, I thought about it at some point (and then got distracted, and then forgot :facepalm:). I've updated.